v-spec.adoc

```diff
@@ -545,20 +545,27 @@ by `vl`, if the value in the `vstart` register is greater than or
 equal to the vector length `vl` then no element operations are
 performed.  The `vstart` register is then reset to zero.
 
-The `vstart` CSR is defined to have only enough writable bits to hold
+The `vstart` CSR is required to have enough writable low bits to hold
 the largest element index (one less than the maximum VLMAX) or
-lg2(VLEN) bits.  The upper bits of the `vstart` CSR are hardwired to
-zero (reads zero, writes ignored).
+lg2(VLEN) bits. The remaining upper bits determine the type
+of index in use.
 
 NOTE: The maximum vector length is obtained with the largest LMUL
 setting (8) and the smallest SEW setting (8), so VLMAX_max = 8*VLEN/8
 = VLEN.  For example, for VLEN=256, `vstart` would have 8 bits to
 represent indices from 0 through 255.
 
+Indexex are represented in "vstart" as either plain text or opaque.
```

Comment:
typo in "indexex"

Reply:
Thank you for the review and correction. 
I'll see what I can do about coaxing Github to let me make the change.